functions-methods.js

```diff
@@ -9,7 +9,16 @@
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
+function getEmailDomain(email) {
+    console.log("Email:",email);
+    const index = email.indexOf("@");
+    const domain = email.substring (index+1, email.length);
```

Comment:
`const domain = email.substring(index+1);` is voldoende. `email.length` is zelfs fout, omdat niet zoveel karakters meer na de `@` voorkomen.